realm/src/main/java/io/realm/RealmQuery.java

```diff
@@ -73,6 +73,7 @@ public RealmQuery(Realm realm, Class<E> clazz) {
         this.realm = realm;
         this.clazz = clazz;
         this.table = realm.getTable(clazz);
+        view = null;
```

Comment:
Personally I like being explicit about these things, so I would just keep it. It is not something I feel strongly about htough. Also It should be `this.view` to follow the naming convention used for the rest of the parameters.

But in general:
```
private boolean bool = false; // I like this better as it is faster to parse mentally.
private boolean bool;  
``